src/clarity-angular/datagrid/datagrid-column.ts

```diff
@@ -43,10 +44,16 @@ export class DatagridColumn extends DatagridFilterRegistrar<DatagridStringFilter
         super(filters);
         this._sortSubscription = _sort.change.subscribe(sort => {
             // We're only listening to make sure we emit an event when the column goes from sorted to unsorted
```

Comment:
It makes complete sense. It still describes exactly what the `if` on the very next line does:
```typescript
if (this.sortOrder !== SortOrder.Unsorted && sort.comparator !== this.sortBy) {
```
We only emit when the column is currently sorted and the provider just switched to a comparator that's not ours. That means another column became sorted, so we're unsorted now.